help/destinations/catalog/personalization/algolia.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Algolia
+description: Use this connector to activate audiences to Algolia for personalization and use across search and recommendations. You can then use the Algolia User Profile source to import the profiles into the Real-Time CDP to build rich audiences.
+---
+
+# [!DNL Algolia] connection
+
+## Overview {#overview}
+
+>[!IMPORTANT]
+>
+>The [!DNL Algolia] destination connector and documentation page are created and maintained by the Algolia Integration Services team. For inquiries or update requests, contact them at [adobe-algolia-solutions@algolia.com](adobe-algolia-solutions@algolia.com).
+
+Use the [!DNL Algolia] destination connection to send Adobe Experience Platform audiences to Algolia to personalize search and recommendations. This requires the use of the [!DNL Algolia User Profiles] source connector to pull user profiles from Algolia to help build rich audiences.
```

Comment:
This part is still unclear to me. Does this destination connector ALWAYS require the prior use of the source connector too? Or can it be used independently? If the source connector is always required, we need to explain the process and also link to the source connector page.